src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPropertySymbol.cs

```diff
@@ -207,5 +208,137 @@ internal override void GenerateMethodBody(TypeCompilationState compilationState,
+
+            public override ImmutableArray<CustomModifier> RefCustomModifiers => _property.RefCustomModifiers;
+
+            public override Symbol AssociatedSymbol => _property;
+
+            public override Symbol ContainingSymbol => _property.ContainingSymbol;
+
+            public override ImmutableArray<Location> Locations => _property.Locations;
+
+            public override Accessibility DeclaredAccessibility => _property.DeclaredAccessibility;
+
+            public override bool IsStatic => _property.IsStatic;
+
+            public override bool IsVirtual => _property.IsVirtual;
+
+            public override bool IsOverride => _property.IsOverride;
+
+            public override bool IsAbstract => _property.IsAbstract;
+
+            public override bool IsSealed => _property.IsSealed;
+
+            public override bool IsExtern => _property.IsExtern;
+
+            public override ImmutableHashSet<string> ReturnNotNullIfParameterNotNull => ImmutableHashSet<string>.Empty;
+
+            internal override bool HasSpecialName => _property.HasSpecialName;
+
+            internal override MethodImplAttributes ImplementationAttributes => MethodImplAttributes.Managed;
+
+            internal override bool HasDeclarativeSecurity => false;
+
+            internal override MarshalPseudoCustomAttributeData? ReturnValueMarshallingInformation => null;
+
+            internal override bool RequiresSecurityObject => false;
+
+            internal override CallingConvention CallingConvention => CallingConvention.HasThis;
+
+            internal override bool GenerateDebugInfo => false;
+
+            public override DllImportData? GetDllImportData() => null;
+
+            internal override ImmutableArray<string> GetAppliedConditionalSymbols()
+                => ImmutableArray<string>.Empty;
+
+            internal override IEnumerable<SecurityAttribute> GetSecurityInformation()
+                => Array.Empty<SecurityAttribute>();
+
+            internal override bool IsMetadataNewSlot(bool ignoreInterfaceImplementationChanges = false) => false;
+
+            internal override bool IsMetadataVirtual(bool ignoreInterfaceImplementationChanges = false) => false;
+
+            internal override bool SynthesizesLoweredBoundBody => true;
+
+            internal override void GenerateMethodBody(TypeCompilationState compilationState, DiagnosticBag diagnostics)
+            {
+                // Method body:
+                //
+                // {
+                //      this.<>backingField = value;
+                // }
+
+                var F = new SyntheticBoundNodeFactory(this, this.GetNonNullSyntaxNode(), compilationState, diagnostics);
+
+                F.CurrentFunction = this;
+                F.CloseMethod(F.Block(
+                    F.Assignment(F.Field(F.This(), _property.BackingField), F.Parameter(Parameters[0])),
+                    F.Return()));
```

Comment:
>F.Return() [](start = 20, length = 10)

nit: do we need the Return?

Reply:
We should, if we're synthesizing the body... all methods need to end with a `ret`.